llvm/lib/Target/SPIRV/MCTargetDesc/SPIRVInstPrinter.cpp

```diff
@@ -242,6 +242,37 @@ void SPIRVInstPrinter::printInst(const MCInst *MI, uint64_t Address,
           }
           break;
         }
+        case SPIRV::OpSubgroupMatrixMultiplyAccumulateINTEL: {
+          const unsigned NumOps = MI->getNumOperands();
+          if (NumFixedOps == NumOps)
+            break; // No extra operands, so no flags to process
+          OS << ' ';
+          // Extract the last operand only if it exists
+          if (NumOps > NumFixedOps) {
```

Comment:
Do we need this check? I'd suggest to remove it and either:
add an assertion or fatal error that NumOps is never < NumFixedOps
or
merge this check with the check on line 247